### ISSUE TITLE
Master branch is broken

### ISSUE THREAD

**Issue by msettles** [dstreett/FLASH2#3]
2 issues at least, first is a ton of what look like debugging print statement are output to screen, second produces a non-valid read that breaks everything.

R just dies, says it expects a blank line
my python app says
a read has more quality values than read characters.

**Comment by msettles** [comment]
Had to go back to the previous commit March 1, 2015 for it to work, this version contains the bug with respect to complete overlaps that are completely within the read, I remember something on this?

**Comment by dstreett** [comment]
What kind of non-valid read are you seeing? Is this preprocess? I ran a
combination of both SE and PE reads through the pipeline and I'm not seeing
any invalid reads. Is this under Brice's directory?

**Comment by msettles** [comment]
I don't run anything on the Idaho machines anymore, this is from pulling
down the head on github. Is github the most up to date?

**Comment by dstreett** [comment]
Yes, I pushed a copy and paste bug. Testing now- making sure the change
doesn't break anything else.

**Comment by dstreett** [comment]
Would you mind checking out the development branch? I fixed the biggest
issue and removed the print statements.

**Comment by msettles** [comment]
Out on the town, but as soon as soon as I get back home

**Comment by dstreett** [comment]
Cool, thanks, Matt.

**Comment by msettles** [comment]
FLASH2 develop compiles are run properly, reads are good. 

Statistics were the same as the March version (which they should be correct?). However the extended reads look more accurate to me than before, though there are 2 reads out of 19,298 which aren’t merged by my expectations. Specifically in this study every Read 1 begins with a TA (after adapter trimming), however since we define anything prior to the start of read 1 as adapter (correct?), then every reads post FLASH merged or not should still begin with a TA (these 2 do not).  Turns out to be completely legitimate overlap and in this case the second base is in disagreement and read2 base is chosen, though I think its the lower quality base (See attachment). Need a second (or third) set of eyes, but here are two reads

@J00113:93:H5GNNBBXX:6:1101:26758:1894_TCGGAACA|Linker_Sol_AP1-3_0|GAAGCAAAT|Primer_Sol_Mar_1b_0_Adapter_2.2_Bar_B*1 1:N:0:TCGGAACA
TACACCGGCGCCGCCGGCACCACCCGGACCGCCGGCAAAACCCTACCCGCCTTGCTTCCACGAACCACCGCGGCCA
+
AFFJJJJJJFJJJJJJ-7AJJ-FJ<-7-AA-AA<A-<---7---<F<-7--------------<AF--A--7-<--

@J00113:93:H5GNNBBXX:6:1101:26758:1894_TCGGAACA|Linker_Sol_AP1-3_0|GAAGCAAAT|Primer_Sol_Mar_1b_0_Adapter_2.2_Bar_B*1 2:N:0:TCGGAACA
GCCGGGGGCGCCGGTGGTGTCGGCGGCGCCGGGGGAACAGGTGGGCTGA
+
JJJJJJJJJJJJJJJJJJJAFJJFJ7AJJJJJ-A-7-7-7<A-7AFFJA

@J00113:93:H5GNNBBXX:6:1101:22079:20304_TCGGAACA|Linker_Sol_AP1-1_0|CCATAACAC|Primer_Sol_Mar_5b_0_Adapter_2.2_Bar_B*0 1:N:0:TCGGAACA
TAAGTGGTGGTCGCGGCGGGGCCGGCGGTAACGCCGGCACGTTCTACGGTTCCGGCGGCGCCGGCGGCGCCG
+
AJJJJJFJJFJJJJFJJJJJFFFFJJJJFJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJ

@J00113:93:H5GNNBBXX:6:1101:22079:20304_TCGGAACA|Linker_Sol_AP1-1_0|CCATAACAC|Primer_Sol_Mar_5b_0_Adapter_2.2_Bar_B*0 2:N:0:TCGGAACA
AGGCGTTGCCGCCGTTACCGCCGCTACCACCCTTTCCTGCTATCGTGGCACCGC
+
JFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFJFFFJJJ<JAJJJFJ

The flashed result
GAAGTGGTGGTCGCGGCGGGAACGGCGGCAACGCCG

I can live with the 2 for sure and add another check for that TA downstream (or I for this work I may increase the min outie overlap).
Matt

**Comment by msettles** [comment]
don’t forget to update the version number
